src/lib/components/commons/FilterPanel/FilterPanel.styled.ts

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import { StyledButton } from '../Button/Button.styled';
```

Comment:
in case we need to import something from another component , then prefer to move it to the index file of that component and then import from "../Button" rather then going into the component folder

Reply:
fix it